pages/dkp/konvoy/2.2/architecture/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: layout.pug
+navigationTitle: Architecture
+title: Concepts and architecture
+menuWeight: 30
+excerpt: Learn the key concepts and architectural components of a Konvoy cluster
+beta: false
+enterprise: false
+---
+
+Kubernetes provides the foundation of a DKP cluster.
+Because of this fundamental relationship, you should be familiar with a few key concepts and terms.
+The topics in this section provide a brief overview of the native Kubernetes architecture, a simplified view of the DKP architecture - both Essential and Enterprise versions. Also, it shows the operational workflow for a DKP cluster.
+
+The following diagram provides a simplified architectural overview to help you visualize the flow of the key components:
+
+![Architectural overview](../img/DKP2ArchDiag.png)
+
+
+Figure 1 - Architectural overview
+
+
+## Master components for the Kubernetes control plane
```

Comment:
Please remove all occurrences of "master" and replace it with control plane, the term "master" is not used in upstream Kubernetes docs